CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace CanHazFunny
+{
+    public class Jester
+    {
+        private readonly IJokeService _jokeService;
+        private readonly IJokeDisplay _jokeDisplay;
+
+        public Jester(IJokeDisplay? jokeDisplay, IJokeService? jokeService)
+        {
+            if (jokeService == null) { throw new System.ArgumentNullException(nameof(jokeService)); }
+            if (jokeDisplay == null) { throw new System.ArgumentNullException(nameof(jokeDisplay)); }
+
+            _jokeService = jokeService;
+            _jokeDisplay = jokeDisplay;
+        }
```

Comment:
```suggestion
            _JokeService = jokeService ?? throw new System.ArgumentNullException(nameof(jokeService));
            _JokeDisplay = jokeDisplay ?? throw new System.ArgumentNullException(nameof(jokeDisplay));
        }
```
You can do this all on the same line with the `??` operator